loaders-app/src/components/HomePage.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import NavBar from './NavBar';
 import SpinnerList from './SpinnerList';
+import '../styles/NavBar.css';
 import '../styles/Spinner.css';
 
 const spinners = [{prop: 'exampleSpinnerProp'}, 
   {prop: 'exampleSpinnerProp'}, 
   {prop: 'exampleSpinnerProp'}];
 
-export const HomePage = () => (
-
+export default () => (
```

Comment:
This is interesting, I think we can just do either: const Homepage = ..., and then export default HomePage at the bottom, or do export default Homepage = ... and no export at the bottom.  Or are you doing some tricky javascript thing here that I don't understand?

Reply:
This is a format that was used in the React course that I took...Basically you can export an anonymous function as the default export from a file since it's not a named export. BUT, let's agree to do your suggestion - const Homepage = ..., and then export default HomePage at the bottom - from here on out. I'll go ahead and change the HomePage file to this format.